src/constants/docsComponentMapping.js

```diff
@@ -63,7 +63,7 @@ export const components = {
       }
     }
   `,
-  h3: styled(TextComponents.LinkedH3)`
+  h3: styled(TextComponents.H3)`
```

Comment:
`<LinkedH../>` elements got `display: inline-block` which caused it it not break. Since headers after H2 do not get linked, I removed it.